app/controllers/facilities_controller.rb

```diff
@@ -4,8 +4,10 @@ class FacilitiesController < ApplicationController
   skip_before_action :authenticate
 
   def validate_params
-    raise ArgumentError unless params[:bbox]&.length == 4
-    params[:bbox].each { |x| Float(x) }
+    if params[:bbox]
+      raise ArgumentError unless params[:bbox]&.length == 4
+      params[:bbox].each { |x| Float(x) }
+    end
```

Comment:
It's not a model field, it's a search parameter, so I wanted to validate and error out in the controller, thinking that malformed requests should be handled in the controller. 

This part of "cleanup" is cleaning up the externally visible behavior - returning an empty list instead of 400 error for missing bbox.